src/extern/bpf/btf.h

```diff
@@ -0,0 +1,334 @@
+/* SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause) */
```

Comment:
ditto: there is a same named header file in this repo.

Reply:
Correct, same as my previous comment, there are differences